Microsoft.Toolkit/Extensions/StringExtensions.cs

```diff
@@ -13,7 +13,7 @@ namespace Microsoft.Toolkit.Extensions
     /// </summary>
     public static class StringExtensions
     {
-        internal const string PhoneNumberRegex = @"^\s*\+?\s*([0-9][\s-]*){9,}$";
+        internal const string PhoneNumberRegex = @"(\([0-9]{3}\) ?)[0-9]{3}-[0-9]{4}|[0-9]{3}-?[0-9]{3}-?[0-9]{4}|([00|+][0-9]{1,2}) ?[0-9]{3}-?[0-9]{3}-?[0-9]{4}";
```

Comment:
Doesn't work for standard US phone numbers with the country code
`19876543210` 

Readable format: `1-987-654-3210`
I think this should be covered if we're changing the format

Reply:
![image](https://user-images.githubusercontent.com/6171200/42695770-5090fc28-8684-11e8-8a9c-1f703b769a20.png)

![image](https://user-images.githubusercontent.com/6171200/42695779-5a74ac80-8684-11e8-9eb9-2df25a5a526d.png)

Works in both cases.

Reply:
It returns a valid phone number true, but if you get the match it would be the first nine digits. It also reports that `1970371774445454545454545455454545454454545` is a valid phone number

Reply:
i see what you mean. Let me take a look.